llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -295,15 +296,95 @@ saveResultModules(std::vector<std::unique_ptr<Module>> &ResModules) {
   return Res;
 }
 
+// Each fallback device library corresponds to one bit in "require mask" which
```

Comment:
As I mentioned in the comments to the description, this additional mapping if library names to bits should better be avoided if there is no compelling performance benefit, as it obscures the logic.

Reply:
I can see now that numeric identification of  libraries have existed before - the `DeviceLibExt` enum in program_manager.hpp. So I withdraw the above review comment - too much to rework.